riptide-stream/src/main/java/org/zalando/riptide/stream/StreamSpliterator.java

```diff
@@ -86,4 +86,13 @@ public long estimateSize() {
     public int characteristics() {
         return ORDERED | IMMUTABLE;
     }
+
+    @Override
+    public void close() throws UncheckedIOException {
```

Comment:
JsonParser will close the stream correctly per default, we may go with this solution. Wrapping it in the Spliterator was the idea to handle the IOException problem, that is not allowed to appear on the stream.